merchantapi/src/main/java/fi/hel/verkkokauppa/merchant/model/MerchantModel.java

```diff
@@ -1,14 +1,30 @@
 package fi.hel.verkkokauppa.merchant.model;
```

Comment:
Same comments than in ConfigurationModel.java regarding the ToString and public fields.